src/openrct2/interface/Colour.h

```diff
@@ -65,7 +67,7 @@ enum
     COLOUR_LIGHT_WATER = 10
 };
 
-enum : uint8_t
+enum : PaletteIndex
 {
     PALETTE_INDEX_0 = 0,     // Transparent
     PALETTE_INDEX_10 = 10,   // Black (0-dark), Dark grey (0)
```

Comment:
Seems like some (if not all, haven't checked) of these became unused. Should we clean it up?

Reply:
I don’t think so. I would actually argue that we should create constants for all palette indices, with correct naming.